src/main/java/WebRTCConf.java

```diff
@@ -0,0 +1,244 @@
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+import java.util.logging.Level;
+
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.edge.EdgeOptions;
+import org.openqa.selenium.firefox.FirefoxOptions;
+import org.openqa.selenium.firefox.FirefoxProfile;
+import org.openqa.selenium.safari.SafariOptions;
+
+
+
+public class WebRTCConf {
+   
+   public static int getRoomID() {
+      Random random = new Random();
+        int roomId = random.nextInt(500000);
+        roomId += 100000;
+        
+        return roomId;
+   }
+   
+   public static ChromeOptions getFireFoxConfiguration() {
+      
+      FirefoxProfile firefoxProfile = new FirefoxProfile();
+        firefoxProfile.setPreference("media.navigator.streams.fake", true);
+        FirefoxOptions firefoxOptions = new FirefoxOptions();
+        firefoxOptions.setProfile(firefoxProfile);
+        
+        ChromeOptions options = new ChromeOptions();
+        options.merge(firefoxOptions);
+        options.setCapability("media.navigator.streams.fake", true);
+
+
+        options.setCapability("browser", "Firefox");
+        options.setCapability("browser_version", "latest");
+        options.setCapability("os", "Windows");
+        options.setCapability("os_version", "10");
+        options.setCapability("build", "WebRTC Dummy Video Call Build - Chrome-FireFox");
+        options.setCapability("name", "WebRTC Room Joining - FireFox Browser");
+        options.setCapability("browserstack.idleTimeout", 300);
+        
+        return options;
+   }
+   
+   public static ChromeOptions getChromeConfiguration() {
+      ChromeOptions options = new ChromeOptions();
+
+        options.addArguments(
+              "--use-fake-device-for-media-stream",
+                "--use-fake-ui-for-media-stream"
+        );
+
+        options.setCapability("browser", "Chrome");
+        options.setCapability("browser_version", "latest");
+        options.setCapability("os", "Windows");
+        options.setCapability("os_version", "10");
+        options.setCapability("build", "WebRTC Dummy Video Call Build - Chrome-FireFox");
+        options.setCapability("name", "WebRTC Room Creation - Chrome Browser");
+        options.setCapability("browserstack.idleTimeout", 300);
+        
+        return options;
+   }
+
+    public static ChromeOptions getEdgeConfiguration() {
+
+        EdgeOptions edgeOptions = new EdgeOptions();
+
+        ChromeOptions options = new ChromeOptions();
+        options.merge(edgeOptions);
```

Comment:
There is currently nothing in edgeOptions methods explicitly added? Is this needed?

Reply:
done